### DISCUSSION TITLE
How to clear handlers to unknown lspconfig servers?

### DISCUSSION THREAD

**Discussion by robertsawko** [mason-org/mason.nvim#1790] [Q&A] [answered]
After a recent update of Mason, I started receiving a warning like this, when opening C++ files
```
mason-lspconfig.setup_handlers: Received handler for unknown lspconfig server name: tsserver.
```
I am suspecting that tsserver is [TypeScript standalone server](https://github.com/microsoft/TypeScript/wiki/Standalone-Server-%28tsserver%29), but I am not too sure? I don't believe I ever installed it myself, but maybe somewhere in my config or some cache files a reference to an old config exists.

I can't find any reference to tsserver in either `LspLog` or `MasonLog`. Please let me know.

**Comment by adibfirman** [comment, marked as answer]
I do follow this answer
https://github.com/williamboman/mason.nvim/issues/1784#issuecomment-2336761735

Seems we need to rename the `tsserver` to `ts_ls` based on [this change](https://github.com/neovim/nvim-lspconfig/pull/3232)
